main.py

```diff
@@ -12,6 +12,9 @@
 from requests import post
 from slackclient import SlackClient
 
+EXCLUDE_USERS = [
+    'UC2PXG134'
```

Comment:
We should maybe put these in some sort of configuration file, since other folks are starting to use this bot on their own workspaces and potentially Slack user IDs could collide maybe? I also think including their username in a comment is useful, in case folks ask to get added/removed.